source/includes/connection-snippets/x509-snippets/ruby-connection.rb

```diff
@@ -0,0 +1,14 @@
+# begin x509 connection
+require 'mongo'
+
+begin
```

Comment:
The begin/end statements in this snippet are unnecessary as they do not do anything.